packages/tabs/addon/components/nucleus-tabs/tab-panel.js

```diff
@@ -0,0 +1,112 @@
+import Component from '@ember/component';
+import { classNames, attributeBindings, classNameBindings, tagName, layout as templateLayout } from '@ember-decorators/component';
+import layout from '../../templates/components/nucleus-tabs/tab-panel';
+import { get, computed } from '@ember/object';
+import defaultProp from '@freshworks/core/utils/default-decorator';
+
+/**
+  __Usage:__
+
+  [Refer component page](/docs/components/nucleus-tabs)
+
+  @class Nucleus Tab Panel
+  @namespace Components
+  @extends Ember.Component
+  @public
+*/
+@tagName('div')
+@templateLayout(layout)
+@classNames('nucleus-tabs__panel')
+@classNameBindings('isActive:is-active')
+@attributeBindings('tabindex')
+@attributeBindings('role')
+@attributeBindings('aria-labelledby')
```

Comment:
`@attributeBindings` accepts a list of values. So this can be refactored to a single line.

Reply:
Done.